nixos/tests/installer.nix

```diff
@@ -291,7 +290,7 @@ let
           # builds stuff in the VM, needs more juice
           virtualisation.diskSize = 8 * 1024;
           virtualisation.cores = 8;
-          virtualisation.memorySize = 1536;
+          virtualisation.memorySize = 2048;
```

Comment:
I had to increase the memory because the test was crashing while doing something with the NixOS manual.
I'm not sure if it's related to this specific test or more general

Reply:
It's unrelated: https://github.com/NixOS/nixpkgs/pull/210930